dbt-cta/spoke_mo/models/1_cta_incremental/_1_cta_incremental__models.yml

```diff
@@ -0,0 +1,1131 @@
+version: 2
+
+models:
+
+  - name: tag_canned_response_base
+    description: ''
+    columns:
+      - name: canned_response_id
+        description: ''
+      - name: tag_id
+        description: ''
+      - name: created_at
+        description: ''
+      - name: id
+        description: ''
+        tests:
+            - not_null
+            - unique
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_tag_canned_response_hashid
+        description: ''
+        tests:
+            - not_null
+            - unique
+
+  - name: zip_code_base
+    description: ''
+    columns:
+      - name: zip
+        description: ''
+        tests:
+            - not_null
```

Comment:
Does this mean theres only one entry in this table per zip code?

Reply:
yep! the zip code file is the only full refresh file. so the zip is the primary key for that file. if there are dupes it suggests we didn't get the full refresh correctly.

Reply:
gotcha!